apps/address-service/package.json

```diff
@@ -4,15 +4,15 @@
   "private": true,
   "scripts": {
     "test": "jest",
-    "dev": "keystone dev --port 3001",
-    "debug": "node inspect node_modules/.bin/keystone dev --port 3001",
+    "dev": "keystone dev",
```

Comment:
Is that service become required to be up for correct work of main application? If this is true I think main service should also start address-service before itself

Reply:
I'll add a command if there are troubles.